fiat-roles/src/main/java/com/netflix/spinnaker/fiat/permissions/DefaultPermissionsResolver.java

```diff
@@ -90,14 +95,28 @@ public UserPermission resolveAndMerge(@NonNull ExternalUser user) {
     return getUserPermission(user.getId(), combo);
   }
 
+  private List<String> trimAndLowerCase(List<String> rolesList) {
+    return rolesList.stream()
```

Comment:
The better place to do this is probably in the `FiatAdminConfig.AdminRoles` class. There are two ways to do this but only 1 might work:

1.) Override the `setAdminRoles(List list)` method to sanitize the input. This would be the ideal case, but unfortunately when Spring injects the values, it doesn't use this method - it uses `List list = getAdminRoles(); list.add(...); list.add(...);` etc

2.) Make `FiatAdminConfig` implement `InitializationBean` and implement the sanitization in the `afterPropertiesSet` method. I don't know if this will work in practice, though. Let me know what you find out. We may just have to keep this method here and go with it.